packages/core/src/generators/react.ts

```diff
@@ -57,19 +57,21 @@ const blockToReact = (json: JSXLiteNode, options: ToReactOptions) => {
   const children = json.children.filter(filterEmptyTextNodes);
 
   if (json.name === 'For') {
-    str += `{${processBinding(json.bindings.each as string, options)}.map(${
+    str += `<>{${processBinding(json.bindings.each as string, options)}.map(${
       json.bindings._forName
     } => (
-      ${children.length === 1 ? '' : '<>'}
+      ${children.length === 1 ? '' : ''}
```

Comment:
why this change? the goal is to do proper idiomatic code

{foo.map(item => <OneChild>...</OneChild />)}

{foo.map(item => <><FirstChild /><SecondChild></>)}

did the old code, as is, have problems?

Reply:
My bad, this is a mistake, should be something like
```suggestion
      ${children.length === 1 ? '' : '<>'}
```